.pack.mk

```diff
@@ -22,17 +22,25 @@ ifeq (${VERSION}, $(filter ${VERSION}, 1.10))
 endif
 
 prepare:
-	rm -rf build packpack
-	git clone https://github.com/packpack/packpack.git
+#	rm -rf build packpack
+#	git clone https://github.com/packpack/packpack.git
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
 		if [ -n "${GIT_TAG}" ]; then \
 			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
 		else \
-			export VERSION="$$(echo ${GIT_DESCRIBE} | sed ${SED_REPLACE_VERSION_REGEX} | sed 's/-/~/').dev"; \
+			if [ "${OS}" = "alpine" ]; then \
+				if git describe --abbrev=0 | grep -q "entrypoint" ; then \
+					export VERSION="$$(echo ${GIT_DESCRIBE} | cut -d- -f1)_alpha0" ; \
```

Comment:
Could you please describe why underscore (not hyphen) before `alpha0`? From docs, I can see `Alpha, release candidates (_rc2), etc are prefixed with underscore _ not a hyphen.`. It would be good to pin it here.

Reply:
Ok